index.js

```diff
@@ -132,9 +134,12 @@ function choo () {
       const _reducers = ns ? reducers[ns] : reducers
       if (_reducers && _reducers[action.type]) {
         if (ns) {
-          const newState = _reducers[action.type](action, state[ns])
-          state[ns] = xtend(state[ns], newState)
-        } else state = xtend(state, reducers[action.type](action, state))
+          const redState = _reducers[action.type](action, state[ns])
```

Comment:
Took me a minute to figure out what you meant by `redState` - perhaps a name like `reducedState` may improve readability?